tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -52,7 +52,10 @@ sub run_test {
         assert_script_run("virsh undefine $guest || virsh undefine $guest --keep-nvram");
         assert_script_run("virsh define $guest.clone");
         assert_script_run("rm -rf $guest.clone");
-        assert_script_run("virt-clone -o $guest -n $guest.clone -f /var/lib/libvirt/images/$guest.clone");
+        record_info "Duplicate a virtual machine from $guest";
```

Comment:
Minor comment, how about replacing duplicate with clone? Others LGTM.

Reply:
sure, have updated this as mentioned.